pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1707,13 +1707,20 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Video).To(HaveLen(devices))
 			Expect(domain.Spec.Devices.Graphics).To(HaveLen(devices))
 
-			if isARM64(arch) && (autoAttach == nil || *autoAttach) {
-				Expect(domain.Spec.Devices.Video[0].Model.Type).To(Equal(v1.VirtIO))
-				Expect(domain.Spec.Devices.Inputs[0].Type).To(Equal(v1.InputTypeTablet))
-				Expect(domain.Spec.Devices.Inputs[1].Type).To(Equal(v1.InputTypeKeyboard))
-			}
-			if isAMD64(arch) && (autoAttach == nil || *autoAttach) {
-				Expect(domain.Spec.Devices.Video[0].Model.Type).To(Equal("vga"))
+			if autoAttach == nil || *autoAttach {
+				switch arch {
+				case "amd64":
```

Comment:
Why isn't this case being tested?
I see that previously `Expect(domain.Spec.Devices.Video[0].Model.Type).To(Equal("vga"))` was expected

Reply:
I screwed up with the refactoring :) Thank you!
Now it is better:
```
case "amd64", "ppc64le":
```